tensorflow/core/kernels/cwise_op_gpu_arg.cu.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+// TODO: Enable GPU support for arg op after resolving
+// build failures on GPU (See #10643 for context).
+#if 0 && GOOGLE_CUDA
```

Comment:
No they haven't been resolved. Since I don't have access to a machine with GPU, I am unable to debug the build failures. Could you suggest a way around this problem?